src/app/TargetSelect/TargetSelect.tsx

```diff
@@ -46,14 +46,9 @@ export interface TargetSelectProps {
   isCompact?: boolean;
 }
 
-interface Target {
-  connectUrl: string;
-  alias: string;
-}
-
 export const TargetSelect: React.FunctionComponent<TargetSelectProps> = (props) => {
   const context = React.useContext(ServiceContext);
-  const [selected, setSelected] = React.useState('');
+  const [selected, setSelected] = React.useState({connectUrl:'', alias:''} as Target);
```

Comment:
And here is the case in point for a `NO_TARGET` constant - the service initializes with `{}` as its default state, but this component uses `{ connectUrl: '', alias: ''}` as its default state - these seem to be intended to mean the same thing, so they should be represented with the same (or at least equal in comparison) value.